servicepulse/intro-pending-retries.md

```diff
@@ -13,6 +13,14 @@ Pending retries messages can be found by going to the pending retries screen.
 
 ![Pending Retries Tab](images/pending-retries.png 'width=500')
 
+### How to enable the screen
+
+Since ServicePulse v1.7.0, the Pending Retries screen is hidden by default. To make it available in the ServicePulse UI, change the following value in `app\js\app.constants.js` to true:
```

Comment:
> `app\js\app.constants.js`

Where do I look for this file? What is that path relative to? e.g. sth like `[SP installation dir]\app\..`